### PR TITLE
[Rec-IM] Activity data fetching from/posting to the API

### PR DESCRIPTION
**Pull request by silasw2**
This is a PR for the working of both the Home page fetching all leagues and displaying their information, the activity page displaying all known info within itself, and the creation of a new league.

### CODE REVIEW

**Comment by silasw2** [comment]
This PR should be ready for merge. I would suggest checking it out and looking at the UI changes as well as the form to see if you have any comments.

File: .env.development [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # REACT_APP_API_URL=https://360Api.gordon.edu/
 
 # @TRAIN
- REACT_APP_API_URL=https://360ApiTrain.gordon.edu/
+# REACT_APP_API_URL=https://360ApiTrain.gordon.edu/
 
 # @LOCALHOST
-# REACT_APP_API_URL=http://localhost:51626/
+REACT_APP_API_URL=http://localhost:51626/
```
Comment (amos-cha) [inline .env.development:10]: Accidentally pushed env file?

File: src/views/RecIM/components/CreateActivityForm/index.js [hunk 2]
```diff
@@ -152,39 +157,37 @@ const CreateActivityForm = ({
...
   }
 
   const handleConfirm = () => {
     setSaving(true);
-    //hard coded for activity, sportId & typeID are hard coded to 0
-    //until we pull API data
-    postSmashActivity().then(() => {
+
+    let activityCreationRequest = { ...currentInfo, ...newInfo };
+    console.log(activityCreationRequest);
```
Comment (amos-cha) [inline src/views/RecIM/components/CreateActivityForm/index.js:177]:
```suggestion
```

File: src/views/RecIM/components/CreateActivityForm/index.js [hunk 3]
```diff
@@ -152,39 +157,37 @@ const CreateActivityForm = ({
...
     return updatedFields;
   }
 
   const handleConfirm = () => {
     setSaving(true);
-    //hard coded for activity, sportId & typeID are hard coded to 0
-    //until we pull API data
-    postSmashActivity().then(() => {
+
+    let activityCreationRequest = { ...currentInfo, ...newInfo };
```
Comment (amos-cha) [inline src/views/RecIM/components/CreateActivityForm/index.js:176]: hot 🔥

**Review by amos-cha — CHANGES_REQUESTED** [review]
Minor changes then it should be good :)

**Review by amos-cha — APPROVED** [review]
Looks Good :)

**Pull request merged**